src/config.rs

```diff
@@ -21,7 +21,7 @@ pub struct Config {
 #[derive(Deserialize, Clone)]
 pub struct Group {
     pub group_name: String,
-    pub cycle_time_ms: u16,
+    pub cycle_time_μs: u16,
```

Comment:
Is u16 big enough if using microseconds? i.e. at most 65 milliseconds cycle time? (previously 65 seconds)

Reply:
good point, thanks

Reply:
FIxed